src/sdk/trace/exporters/stdout_exporter.zig

```diff
@@ -0,0 +1,58 @@
+const std = @import("std");
+const pb = @import("../../../opentelemetry/proto/trace/v1.pb.zig");
+const protobuf = @import("protobuf");
+const SpanExporter = @import("../span_exporter.zig").SpanExporter;
+
+/// GenericWriterExporter is the generic SpanExporter that outputs spans to the given writer.
+fn GenericWriterExporter(
+    comptime Writer: type,
+) type {
+    return struct {
+        writer: Writer,
+
+        const Self = @This();
+
+        pub fn init(writer: Writer) Self {
+            return Self{
+                .writer = writer,
+            };
+        }
+
+        pub fn exportSpans(ctx: *anyopaque, spans: []pb.Span) anyerror!void {
```

Comment:
What is `ctx` and what does it represent?
Is it a generic structure that has no well-defined field?